CityOfDetroit/bloom
# feat: add user csv export

# Pull Request Template

## Issue Overview

This PR addresses #1538


## Description

This adds a new endpoint to the backend that exports all partner users to a csv file. It also adds an "export" button to the users page on the partner site (made the export and add users button the smaller size as well).

The export includes all of the columns referenced here: https://docs.google.com/spreadsheets/d/1RrYNnkAdG4QPyVBC8yG1svqPaLt0V4qVQMzQgHUEIjY/edit#gid=670984196.

## How Can This Be Tested/Reviewed?

When running this locally and clicking on "export" you should get a csv download with all of the users shown on the users page.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
